### PR TITLE
Review and integrate open729

### PR DESCRIPTION
**Pull request by charlesh88**
@akhenry Relates to #729, #707 and #498. Needed by work done in open707 by Andrew. Can be integrated directly into master prior to merge into open707 if desired with no adverse effects.

Subtle but significant mods via markup and CSS changes to elements related to configuration options for plot and table object types, and holder elements in the Inspector.

### Author Checklist

1. Changes address original issue? Y
2. Unit tests included and/or updated with changes? N, N/A
3. Command line build passes? Y
4. Changes have been smoke-tested? Y

### CODE REVIEW

**Comment by charlesh88** [comment]
@akhenry Any chance you'd be able to review and merge this by EOD today? There's some stuff in this I'd love to get into the testathon for tomorrow. You don't need to integrate this into 707 if you don't want to. Thanks!

**Comment by akhenry** [comment]
@charlesh88 yes of course, thanks for bringing it to my attention, had slipped off my radar. I'll take a look now.

**Comment by charlesh88** [comment]
@akhenry YDM, thanks!

File: gulpfile.js [hunk 1]
```diff
@@ -60,7 +60,8 @@ var gulp = require('gulp'),
             singleRun: true
         },
         sass: {
-            includePaths: bourbon.includePaths
+            includePaths: bourbon.includePaths,
+            sourceComments: true
```
Comment (akhenry) [inline gulpfile.js]: Is this meant to be here in production?

File: platform/features/plot/src/PlotOptionsForm.js [hunk 2]
```diff
@@ -110,7 +112,8 @@ define(
                             {
                                 'name': 'Markers',
                                 'control': 'checkbox',
-                                'key': 'markers'
+                                'key': 'markers',
+                                'layout': 'control-first'
```
Comment (akhenry) [inline platform/features/plot/src/PlotOptionsForm.js]: :+1:

File: gulpfile.js (hunk 1 above)
Comment (charlesh88) [inline gulpfile.js]: Yes, for now. According to Pete, this won't affect performance and it's very useful to me for debugging.

Comment (larkin) [inline gulpfile.js]: Yeah-- we're not incredibly concerned about size of static assets and considering the inconsistencies with sourcemap support in chrome this is the next best option.  If chrome starts displaying sourcemaps, then we can turn this off.  (I can't find a reason why they don't work; they work in firefox just fine).

Comment (akhenry) [inline gulpfile.js]: Ok, no worries, just checking.

File: platform/commonUI/general/res/sass/forms/_elems.scss [hunk 3]
```diff
@@ -37,12 +31,21 @@
...
+        padding: $formTBPad $formLRPad;
+    }
+
 	.form-row {
 		$m: $interiorMargin;
 		box-sizing: border-box;
 		@include clearfix;
 		border-top: 1px solid $colorFormLines;
-		margin-top: $m;
+		//margin-top: $m;
```
Comment (akhenry) [inline platform/commonUI/general/res/sass/forms/_elems.scss]: Commented code, should be removed

File: platform/commonUI/general/res/sass/forms/_elems.scss [hunk 4]
```diff
@@ -52,9 +55,9 @@
 		>.label,
 		>.controls {
 			box-sizing: border-box;
-			@include clearfix;
+			//@include clearfix;
```
Comment (akhenry) [inline platform/commonUI/general/res/sass/forms/_elems.scss]: Commented code

File: platform/commonUI/general/res/sass/forms/_elems.scss [hunk 5]
```diff
@@ -52,9 +55,9 @@
...
 			font-size: 0.8rem;
-			line-height: $formInputH;
+			//line-height: $formInputH;
```
Comment (akhenry) [inline platform/commonUI/general/res/sass/forms/_elems.scss]: Commented code

**Comment by akhenry** [comment]
@charlesh88 Looks good, just a couple of lines of commented code that should probably be removed

File: platform/commonUI/general/res/sass/forms/_elems.scss (hunk 5 above)
Comment (charlesh88) [inline platform/commonUI/general/res/sass/forms/_elems.scss]: One sec...

**Comment by charlesh88** [comment]
@akhenry Just pushed an updated open729 that kills those lines that were commented out. Sorry about that!

**Comment by akhenry** [comment]
### Reviewer Checklist

1. Changes appear to address issue? __Y__
2. Appropriate unit tests included? __N/A__ - markup and style changes, does not necessitate unit tests
3. Code style and in-line documentation are appropriate? __Y__
4. Commit messages meet standards? __Y__

**Comment by charlesh88** [comment]
Thanks!

**Pull request merged**